web/components/toolbar.tsx

```diff
@@ -11,8 +12,8 @@ interface IToolbar {
 }
 
 const Toolbar: React.FC<IToolbar> = ({
-  toQuicklook,
-  toGlobalization,
+  toQuicklook = '#',
+  toGlobalization = '#',
   children,
```

Comment:
I provided default values to links because the `href` prop of the `Link` component is typed strongly - it doesn’t accept `undefined`, different from `a`.

Reply:
(In case somebody asks)